dashboards/haproxy/haproxy-prometheus.json

```diff
@@ -437,23 +520,154 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "sum by (instance) (rate(haproxy_backend_response_errors_total{${Cluster},${Location},${Namespace}}[10m]))"
+                  "prometheusQuery": "sum by (instance) (rate(haproxy_backend_response_errors_total{${Cluster},${Location},${Namespace}}[10m]))",
+                  "unitOverride": ""
                 }
               }
             ],
+            "thresholds": [],
             "timeshiftDuration": "0s",
             "yAxis": {
+              "label": "",
               "scale": "LINEAR"
             }
           }
-        },
-        "width": 6,
-        "xPos": 6,
-        "yPos": 15
+        }
+      },
+      {
+        "yPos": 8,
+        "width": 24,
+        "height": 4,
+        "widget": {
+          "title": "Frontends",
+          "text": {
+            "content": "",
+            "format": "MARKDOWN",
+            "style": {
+              "backgroundColor": "#FFFFFF",
+              "fontSize": "FS_LARGE",
+              "horizontalAlignment": "H_LEFT",
+              "padding": "P_EXTRA_SMALL",
+              "pointerLocation": "POINTER_LOCATION_UNSPECIFIED",
+              "textColor": "#212121",
+              "verticalAlignment": "V_TOP"
+            }
+          }
+        }
+      },
+      {
+        "xPos": 24,
+        "yPos": 8,
+        "width": 24,
+        "height": 4,
+        "widget": {
+          "title": "Backends",
+          "text": {
+            "content": "",
+            "format": "MARKDOWN",
+            "style": {
+              "backgroundColor": "#FFFFFF",
+              "fontSize": "FS_LARGE",
+              "horizontalAlignment": "H_LEFT",
+              "padding": "P_EXTRA_SMALL",
+              "pointerLocation": "POINTER_LOCATION_UNSPECIFIED",
+              "textColor": "#212121",
+              "verticalAlignment": "V_TOP"
+            }
+          }
+        }
+      },
+      {
+        "width": 12,
+        "height": 8,
+        "widget": {
+          "title": "Frontend 2XX %",
+          "scorecard": {
+            "blankView": {},
+            "thresholds": [],
+            "timeSeriesQuery": {
+              "outputFullDuration": true,
+              "prometheusQuery": "(sum(rate(haproxy_frontend_http_responses_total{${Cluster},${Location},${Namespace}[1m]),code=\"2xx\"}) / sum(rate(haproxy_frontend_http_responses_total{${Cluster},${Location},${Namespace}}[1m]))) * 100",
```

Comment:
This one didn't work when I loaded the dashboard locally - I think the [1m] just needed to be after the 2xx filter. It looks like the 5xx query is good, so this works just swapping in "2xx" instead:

`(sum(rate(haproxy_frontend_http_responses_total{${Cluster},${Location},${Namespace},code=\"2xx\"}[1m])) / sum(rate(haproxy_frontend_http_responses_total{${Cluster},${Location},${Namespace}}[1m]))) * 100`

Reply:
Thanks for catching this!